merlin/systems/dag/op_runner.py

```diff
@@ -18,7 +18,17 @@
 
 
 class OperatorRunner:
-    def __init__(self, config, repository="./", version=1, kind=""):
```

Comment:
removed `kind` since it doesn't appear to be used currently. Can be added back if required in future